README-ocp4-cs.md

```diff
@@ -0,0 +1,388 @@
+  - [Create the two OpenShift clusters](#create-the-two-openshift-clusters)
+  - [Configure client context for cluster admin access](#configure-client-context-for-cluster-admin-access)
+  - [Deploy Federation](#deploy-federation)
+  - [Register the clusters](#register-the-clusters)
+- [Example application](#example-application)
+  - [Deploy the application](#deploy-the-application)
+  - [Verify that the application is running](#verify-that-the-application-is-running)
+  - [Modify placement](#modify-placement)
+- [Clean up](#clean-up)
+- [What’s next?](#whats-next)
+
+<!-- /TOC -->
+
+<a id="markdown-introduction" name="introduction"></a>
+# Introduction
+
+This demo is a simple deployment of [KubeFed
+Operator](https://operatorhub.io/operator/kubefed-operator) on two OpenShift
+clusters. A sample application is deployed to both clusters through the
+KubeFed controller.
+
+<a id="markdown-pre-requisites" name="pre-requisites"></a>
+# Prerequisites
+
+The Federation Operator requires at least one [OpenShift Container Platform](https://www.openshift.com/) 4.1 cluster.
+
+This walkthrough will use two OCP 4.1 clusters deployed using the [Installer-Provisioned Infrastructure](https://cloud.redhat.com/openshift/install/aws) installation type.
+
+<a id="markdown-install-the-kubefedctl-binary" name="install-the-kubefedctl-binary"></a>
+## Install the kubefedctl binary
+
+The `kubefedctl` tool manages federated cluster registration. Download the
+v0.1.0-rc3 release and unpack it into a directory in your PATH (the
+example uses `$HOME/bin`):
+
+~~~sh
+curl -Ls https://github.com/kubernetes-sigs/kubefed/releases/download/v0.1.0-rc3/kubefedctl-0.1.0-rc3-linux-amd64.tgz | tar xz -C ~/bin
+~~~
+
+Verify that `kubefedctl` is working:
+
+~~~sh
+kubefedctl version
+
+kubefedctl version: version.Info{Version:"v0.1.0-rc3", GitCommit:"d188d227fe3f78f33d74d9a40b3cb701c471cc7e", GitTreeState:"clean", BuildDate:"2019-06-25T00:27:58Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"linux/amd64"}
+~~~
+
+<a id="markdown-download-the-example-code" name="download-the-example-code"></a>
+## Download the example code
+
+Clone the demo code to your local machine:
+
+~~~sh
+git clone https://github.com/openshift/federation-dev.git
+cd federation-dev/
+~~~
+
+<a id="markdown-federation-deployment" name="federation-deployment"></a>
+# Federation deployment
+
+<a id="markdown-create-the-two-openshift-clusters" name="create-the-two-openshift-clusters"></a>
+## Create the two OpenShift clusters
+
+Follow the [Installer Provisioned Infrastructure Instructions](https://cloud.redhat.com/openshift/install/aws/installer-provisioned) for installing two OpenShift 4.1 clusters on AWS. Be sure to employ the `--dir` option with a distinct directory for each cluster you create.
+
+<a id="markdown-configure-client-context-for-cluster-admin-access" name="configure-client-context-for-cluster-admin-access"></a>
+## Configure client context for cluster admin access
```

Comment:
What do you think about referencing your video about flattening kubeconfig files? 

https://www.youtube.com/watch?v=zuh3JFtNbeU